src/constants.js

```diff
@@ -163,3 +163,50 @@ export const FEATURE_TOGGLE_DEFAULTS = {
 
 // Project id configured in https://walletconnect.com
 export const WALLET_CONNECT_PROJECT_ID = '8264fff563181da658ce64ee80e80458';
+
+export const STAGE_DEV_PRIVNET = 'dev-privnet';
+export const STAGE_TESTNET = 'testnet';
+export const NETWORK_TESTNET = 'testnet';
+export const WALLET_SERVICE_TESTNET_BASE_URL = 'https://dev.wallet-service.testnet.hathor.network/';
+export const WALLET_SERVICE_TESTNET_BASE_WS_URL = 'wss://ws.dev.wallet-service.testnet.hathor.network/';
+export const NODE_SERVER_TESTNET_URL = 'https://node1.testnet.hathor.network/v1a/';
+export const EXPLORER_TESTNET_URL = 'https://explorer.testnet.hathor.network/';
+
+export const PRE_SETTINGS_TESTNET = {
+  stage: STAGE_TESTNET,
+  network: NETWORK_TESTNET,
+  walletServiceUrl: WALLET_SERVICE_TESTNET_BASE_URL,
+  walletServiceWsUrl: WALLET_SERVICE_TESTNET_BASE_WS_URL,
+  nodeUrl: NODE_SERVER_TESTNET_URL,
+  explorerUrl: EXPLORER_TESTNET_URL,
+};
+
+export const NODE_SERVER_MAINNET_URL = 'https://mobile.wallet.hathor.network/v1a/';
+export const EXPLORER_MAINNET_URL = 'https://explorer.hathor.network/';
+
+export const PRE_SETTINGS_MAINNET = {
+  stage: STAGE,
+  network: NETWORK,
+  walletServiceUrl: WALLET_SERVICE_MAINNET_BASE_URL,
+  walletServiceWsUrl: WALLET_SERVICE_MAINNET_BASE_WS_URL,
+  nodeUrl: NODE_SERVER_MAINNET_URL,
+  explorerUrl: EXPLORER_MAINNET_URL,
+};
+
+/**
+ * Network settings key collection to be used with the app
+ * storage.
+ */
+export const networkSettingsKey = {
```

Comment:
What do you think about renaming this to `networkSettingsKeyMap`?

`networkSettingsKey` implies imo that there is only one key, which is the case now, but might not be true in the future

Reply:
I used `networkSettingsKey` just keep consistency with [`unleashStorageKey`](https://github.com/HathorNetwork/hathor-wallet-mobile/blob/562ee965ce5f0c68e0810fc16b81453893eece5c/src/constants.js#L90) and [`pushNotificationKey`](https://github.com/HathorNetwork/hathor-wallet-mobile/blob/562ee965ce5f0c68e0810fc16b81453893eece5c/src/constants.js#L106). Can you pass this time and I open a new PR just to "fix" all the cases together?

Reply:
I changed the name in this commit on PR2: c9c81e4559d8b63ede0b165a60e5af399809f613

Reply:
Thanks!